modular_skyrat/modules/sec_haul/code/guns/ammo.dm

```diff
@@ -1,35 +1,6 @@
 /obj/item/ammo_box/advanced
 	multiple_sprites = AMMO_BOX_FULL_EMPTY
 
-/datum/techweb_node/smartgun_rails
-	id = "smartgun_rails"
-	display_name = "Experimental SMARTGUN Ammunition"
-	description = "Standard ammo for a non-standard SMARTGUN."
-	prereq_ids = list("electronic_weapons", "exotic_ammo")
-	design_ids = list("smartgun")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 3000)
```

Comment:
😔

Reply:
See reply to the sec_haul guns file comment.